### PR TITLE
[findutils]: Add inspec tests

### PR DESCRIPTION
**Pull request by gavindidrichsen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+
+> pkg_deps=(core/findutils)
+
+### Use as Tool
+
+#### Installation
+
+To install this plan, you should run the following commands to first install, and then link the binaries this plan creates.
+
+``hab pkg install core/findutils --binlink``
```
Comment (MindNumbing) [inline README.md:35]: Can you show the installation / binlinking taking place as it would in the CLI?

File: controls/findutils_exists.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  describe plan_installation_directory do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+    its('stderr') { should be_empty }
+  end
+
+  command_relative_path = input('command_relative_path', value: 'bin/find')
+  command_full_path = File.join(plan_installation_directory.stdout.strip, command_relative_path)
+  describe file(command_full_path) do
+    it { should exist }
```
Comment (MindNumbing) [inline controls/findutils_exists.rb:22]: Can you add a check that the file is an executable?

**Review by MindNumbing — CHANGES_REQUESTED** [review]
A few small changes

File: README.md (hunk 1 above)
Comment (gavindidrichsen) [reply README.md:35]: Added.

File: controls/findutils_exists.rb (hunk 2 above)
Comment (gavindidrichsen) [reply controls/findutils_exists.rb:22]: Added.

**Comment by gavindidrichsen** [comment]
Hi @MindNumbing.  Changes added according to comments.  Thanks.

**Review by MindNumbing — APPROVED** [review]
Thanks @gavindidrichsen, LGTM! 👍

**Pull request merged**